wiremock/mappings/directbidder-test-app.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "directbidder-test-app",
+  "metadata": {
+    "origin": "http://directbidder-test-app.par.preprod.crto.in/inapp/v1?profileId=235"
+  },
+  "request": {
+    "url": "/directbidder-test-app/inapp/v1?profileId=235",
+    "method": "POST"
+  },
+  "response": {
+    "status" : 200,
+    "body" : "{\r\n      \"slots\": [{\r\n        \"impId\": \"5f05ec4e58f360e11c914a9de6650fc4\",\r\n        \"placementId\": \"div-Test-DirectBidder\",\r\n        \"arbitrageId\": \"arbitrage_id\",\r\n        \"cpm\": \"1.00\",\r\n        \"currency\": \"EUR\",\r\n        \"width\": 300,\r\n        \"height\": 250,\r\n        \"ttl\": 0,\r\n        \"displayUrl\": \"https:\/\/directbidder-stubs.par.preprod.crto.in\/delivery\/ajs.php?width=300&height=250\"\r\n      }],\r\n      \"requestId\": \"bffa74c5-c75e-47ff-8ddf-ead9a88f74ae\"\r\n    }",
```

Comment:
Is it possible to really have multiline strings in wiremock mapping files ? Then it could make the response more readable. Else, do you think the the \r\n coudl be skipped ?

Reply:
When I used WireMock in the past, I remember I had to do the same escaping. FWIW, I was also compressing the output (I was using: https://www.onlinejsonconvert.com/json-escape/).

Reply:
JSON format does not support multiline, nor comments ¯\\\_(ツ)_/¯ it could be inlined though, without \\r\\n but as @Amokrane said, it is easy to use an escape tool.